src/main/java/serializers/SlotsSerializer.java

```diff
@@ -0,0 +1,33 @@
+package serializers;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import domain.Slots;
+import java.io.IOException;
+
+/**
+ * Created by jakubr on 09/08/16.
+ */
+public class SlotsSerializer extends StdSerializer<Slots>{
+
+    public SlotsSerializer(){
+        this(null);
+    }
+
+    public SlotsSerializer(Class<Slots> t){
+        super(t);
+    }
+
+    @Override
+    public void serialize(Slots slot, JsonGenerator jgen, SerializerProvider provider) throws IOException, JsonProcessingException{
+        jgen.writeStartObject();
```

Comment:
Is there a test in place that will fail when the structure of the object changes?